include/baresip.h

```diff
@@ -881,6 +886,7 @@ enum answer_method {
 /** Defines the User-Agent event handler */
 typedef void (ua_event_h)(struct ua *ua, enum ua_event ev,
 			  struct call *call, const char *prm, void *arg);
+typedef void (bevent_h)(struct bevent *event, void *arg);
```

Comment:
suggested signature:

```
typedef void (bevent_h)(struct ua_event ev, struct bevent_payload *event, void *arg);
```

All events must have a valid EV event set.

The payload is optional and type dependent.

Reply:
I'll try this.

Reply:
Do you mean?
```c
typedef void (bevent_h)(enum ua_event ev, struct bevent_payload *event, void *arg);
```

Rename `struct bevent` -> `struct bevent_payload`?